CMR/python/cmr/search/collection.py

```diff
@@ -100,6 +100,7 @@ def apply_filters(filters, items):
     """
     return scom.apply_filters(filters, items)
 
+# document-it: {"from":"cmr.search.common.search_by_page"}
 def search(query=None, filters=None, limit=None, config:dict = None):
```

Comment:
same comment about style consistency

Reply:
it's the style I got from the one public commit that we accepted, when using a type hint, you space the =